index.js

```diff
@@ -496,8 +496,8 @@ class NodeSwordInterface {
    */
   hebrewStrongsAvailable() {
     try {
-      this.nativeInterface.getLocalModule("StrongsHebrew");
-      return true;
+      const strongsDict = this.nativeInterface.getLocalModule("StrongsHebrew");
```

Comment:
Seems like `this.nativeInterface.getLocalModule()` will return `undefined` if module is not found and would not throw any errors

Reply:
Thanks once more for catching this one! I checked the history of changes in node-sword-interface and found that this is indeed a regression. Previously the function getLocalModule was throwing in exception when the respective module was not found, but I changed that at some point (on purpose).